src/opm/parser/eclipse/share/keywords/900_OPM/S/SWOFLET

```diff
@@ -0,0 +1,115 @@
+{
+  "name": "SWOFLET",
+  "sections": [
+    "PROPS"
+  ],
+  "size": {
+    "keyword": "TABDIMS",
+    "item": "NTSFUN"
+  },
+  "requires": ["OIL", "WATER"],
```

Comment:
This is a large and complex keyword, is it organized to ensure that:
1. The most important parameters come first.
2. There are valid defaults - so that the items (on the end) can hopefully be ignored?